bottlecap/src/tags/lambda/tags.rs

```diff
@@ -1,13 +1,16 @@
+use crate::config;
+use log::warn;
```

Comment:
don't use `log` crate

Reply:
ty! My IDE doesn't do the right thing whenever I add new loggers